src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -1232,7 +1233,7 @@ void EnIk_CsAction4(EnIk* this, PlayState* play) {
 }
 
 void EnIk_CsAction5(EnIk* this, PlayState* play) {
```

Comment:
```suggestion
void EnIk_CsCue5(EnIk* this, PlayState* play) {
```
Should these actions change to `cue` now?

Reply:
these dont map to cues, they are "cs action functions" (`EnIkCsAction` enum)